src/components/search/GlobalSearchField.js

```diff
@@ -0,0 +1,142 @@
+/**
+ *
+ * @author Sriram
+ * A component that global search field with options to filter on apps, analyses and data
+ */
+import React from "react";
+import ids from "./ids";
+import { build, formatMessage, withI18N } from "@cyverse-de/ui-lib";
+import intlData from "../layout/messages";
+import NavigationConstants from "../layout/NavigationConstants";
+
+import SearchIcon from "@material-ui/icons/Search";
+import {
+    FormControl,
+    InputAdornment,
+    InputBase,
+    MenuItem,
+    Select,
+} from "@material-ui/core";
+
+import { makeStyles } from "@material-ui/core/styles";
+import { injectIntl } from "react-intl";
+import { useRouter } from "next/router";
+
+const useStyles = makeStyles((theme) => ({
+    search: {
+        position: "relative",
+        borderRadius: theme.shape.borderRadius,
+        backgroundColor: theme.palette.white,
+        "&:hover": {
+            backgroundColor: theme.palette.white,
+        },
+        marginRight: 0,
+        marginLeft: 0,
+        [theme.breakpoints.up("sm")]: {
+            marginLeft: theme.spacing(2),
+            width: "70%",
+        },
+        [theme.breakpoints.down("xs")]: {
+            backgroundColor: theme.palette.bgGray,
+            float: "left",
+            margin: theme.spacing(1),
+            width: "60%",
+        },
+    },
+    searchIcon: {
+        width: theme.spacing(7),
+        height: "100%",
+        position: "absolute",
+        pointerEvents: "none",
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+    },
+    searchFilter: {
+        width: 100,
+        marginRight: theme.spacing(4),
+        paddingLeft: theme.spacing(1),
+        border: 1,
+        backgroundColor: theme.palette.primary.main,
+        color: theme.palette.white,
```

Comment:
Since `backgroundColor` is `theme.palette.primary.main`, then `color` should probably be `theme.palette.primary.contrastText`.